multiplayer/src/services/gameClient.ts

```diff
@@ -22,8 +22,10 @@ import {
     setGameMetadataAsync,
 } from "../epics";
 
-const GAME_HOST = "https://multiplayer.staging.pxt.io";
-//const GAME_HOST = "http://localhost:8082";
+const GAME_HOST_PROD = "https://mp.makecode.com";
```

Comment:
I think for this and the --multiplayer url we should probably toss them into pxt.webConfig, but that's unrelated to this pr.